packages/vscode-zipfs/sources/TerminalLinkProvider.ts

```diff
@@ -21,7 +21,7 @@ const SELECTOR_PATTERNS = [
 ];
 
 const COMBINED_SELECTORS = SELECTOR_PATTERNS.map(selector => `(${selector.source})`).join(`|`);
-const FILEPATH_MATCHER = /(([A-Za-z]:)?\/.*((\$\$virtual)|(\.zip)).*\.[\w:]+)/.source;
+const FILEPATH_MATCHER = /(([A-Za-z]:)?\/.*((__virtual__)|(\.zip)).*\.[\w:]+)/.source;
```

Comment:
```suggestion
const FILEPATH_MATCHER = /(([A-Za-z]:)?\/.*((__virtual__)|(\$\$virtual)|(\.zip)).*\.[\w:]+)/.source;
```
Would prefer if it understood both for a while

Reply:
Indeed - was the intent, forgot to add it here